packages/client/src/lib.js

```diff
@@ -33,6 +34,12 @@ const MAX_CONCURRENT_UPLOADS = 3
 const DEFAULT_CHUNK_SIZE = 1024 * 1024 * 10 // chunk to ~10MB CARs
 const MAX_BLOCK_SIZE = 1048576
 const MAX_CHUNK_SIZE = 104857600
+// These need to match what is configured in the Cloudflare dashboard
+// The limit is 30, but throttledQueue has an off-by-one bug!
+const RATE_LIMIT_REQUESTS = 29
```

Comment:
I think it was actually the test that was counting a request that was released on or after the current period elapsed.

Reply:
Ah yes, so obvious when I look at it the test again now! Thanks.